firefox-ios/nimbus-features/messaging/messaging-evergreen-messages.fml.yaml

```diff
@@ -53,3 +53,22 @@ import:
                 action: OPEN_URL
                 action-params:
                   url: https://www.macrumors.com
+
+              # Serves as an example of how microsurveys may look like
+              microsurvey-message:
+                surface: microsurvey
+                style: MICROSURVEY
+                trigger-if-all:
+                  - "number_of_launches >= 2"
+                title: "Firefox Survey"
+                text: "What is your favorite firefox feature?"
```

Comment:
@tom I'm thinking we remove the text since we don't have a fallback and should avoid showing the text if this is nil in the codebase.

Reply:
Agreed, we don't default any text here either but if this is supposed to just be an example then I think it's worth keeping, wdyt?

Reply:
Makes sense! I added the printing example and a comment for more clarity, let me know if that works!